### PR TITLE
ZKUI-79: add a warning when leaving the page while a update is on going

### PR DESCRIPTION
**Pull request by gaudiauj**

### CODE REVIEW

**Review by ChengYanJin — APPROVED** [review]

File: src/react/databrowser/objects/Objects.jsx [hunk 1]
```diff
@@ -29,6 +29,17 @@ export default function Objects() {
   );
   const listType = useSelector((state: AppState) => state.s3.listObjectsType);
   const bucketInfo = useSelector((state: AppState) => state.s3.bucketInfo);
+  const isUploading = useSelector((state: AppState) => state.networkActivity.messages.includes('Uploading object(s)'));
```
Comment (JBWatenbergScality) [inline src/react/databrowser/objects/Objects.jsx]: nit: Can you extract `Uploading object(s)` to a const ?

File: src/react/actions/constants.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+// @flow
+export const UPLOADING_OBJECT = 'Uploading object(s)';
```
Comment (ChengYanJin) [inline src/react/actions/constants.js]: I would suggest we add the action in `src/react/actions/s3object.js`

**Review by MonPote — APPROVED** [review]

**Comment by ChengYanJin** [comment]
Close the pull request because of the wrong branch name, will open a new PR.

**Pull request closed without merging**